PageSpeed/MainViewController.swift

```diff
@@ -39,18 +39,18 @@ class MainTabBarController: UITabBarController {
 // MARK: - MainViewController
 
 class MainViewController: UIViewController {
-
+    
```

Comment:
@ilyageekhub here is `Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)`

Reply:
fix